test/capture.js

```diff
@@ -1,3 +1,21 @@
+define([ 'intern!tdd',
+        'intern/chai!expect',
+        '../pointerevents',
+        'src/boot',
```

Comment:
If the source files aren't referenced, many of the pep objects aren't initialized.  I should remove ../pointerevents, as it is not having the intended effect, and I don't think we want to copy it into the test directory.